### ISSUE TITLE
Leaflet bindPopup issue

### ISSUE THREAD

**Issue by Mahdi-Roozbahani** [Mahdi-Roozbahani/main#3]
In userinfo line173, 
`marker.bindPopup("This is Project "+pname+"<br><a onclick='return clickedProjectFunction(pname);'>Go to project</a>").openPopup();`

onlcik does not pass the project name, instead it passes undfined variable.

**Comment by sangyh** [comment]
onclick of what? the <a> tag? wasnt this working properly when we checked?

**Comment by Mahdi-Roozbahani** [comment]
this is the onclick='return clickedProjectFunction(pname);' that you used in popup. It works but it does not pass the pname to the "clickedProjectFunction". in fact, it doe not know pname as a variable when you call it like this.

**Comment by Mahdi-Roozbahani** [comment]
I fixed the problem like the below code:

`var link = $('<a href="#" class="speciallink">Go to project Named'+pname+'</a>').click(function() {		clickedProjectFunction(pname);})[0];`

`marker.bindPopup(link).openPopup();`

Note I created a href variable and I called the clickedProjectFunction inside href.click function. Now pname is accessible to this function. Beforehand it was just a string that did not make sense for the function.
